### PR TITLE
Feature: Chappie2.0 route redirects to make old URLs backwards compatible

### PR DESCRIPTION
**Pull request by svenjens**
Instead of creating a catch all. 
This pr takes care of handling to old routes and create a permanent redirect.

### CODE REVIEW

**Review by afjlambert — CHANGES_REQUESTED** [review]
We already have the concept of redirecting.

Why did you not add the following to `export const redirectConfig` in `routes.js`? 
```
["/:slug/locatie", "/:slug"],
["/:slug/adresgegevens", "/:slug"],
["/:slug/vragen", "/:slug"],
["/:slug/uitkomsten", "/:slug"],
["/:slug/conclusie", "/:slug"],
```

I do like `{ from: "/:slug/conclusie", to: "/:slug" },` signature much better then the `['', '']` I introduced before.
What would be really great is `routes.ts` with `export const redirectConfig: { from: string, to: string } = ...`.

And simply change:

```
{redirectConfig.map(([from, to]) => (
  <Redirect key={from} {...{ from, to }} />
))}
```
to
```
{redirectConfig.map((config) => (
  <Redirect key={from} {...config} />
))}
```

**Comment by svenjens** [comment]
@afjlambert This is because we need the slug. I added the redirects to the others before but then we just have a redirect to "/:slug" We need a base route for the slug to work.

File: apps/client/src/components/Router.js [hunk 1]
```diff
@@ -16,9 +16,20 @@ const Router = (props) => {
...
           {routeConfig
             .filter((route) => route.component)
-            .map((route, i) => (
-              <Route key={i} {...route} />
-            ))}
+            .map((route, i) => {
+              if (!route.redirect) {
+                return <Route key={i} {...route} />;
+              } else {
+                return route.redirect.map((redirect, i) => (
```
Comment (robinpiets) [inline apps/client/src/components/Router.js]: Here you don't need the `i`

File: apps/client/src/components/Router.js [hunk 2]
```diff
@@ -16,9 +16,20 @@ const Router = (props) => {
           ))}
           {routeConfig
             .filter((route) => route.component)
-            .map((route, i) => (
-              <Route key={i} {...route} />
-            ))}
+            .map((route, i) => {
```
Comment (robinpiets) [inline apps/client/src/components/Router.js]: Can be refactored to (pseudo code):
```
 .map((route, i) => route.redirect ? route.redirect.map((redirect) => (<MyRedirect key={redirect.from} {...redirect} />) : <Route key={i} {...route} />)
```

**Review by robinpiets — CHANGES_REQUESTED** [review]

**Comment by afjlambert** [comment]
Can you validate if my last commit actually works?

**Review by robinpiets — APPROVED** [review]
LGTM!

Edit:  conflicts 😢

**Pull request merged**